course-ideas.md

```diff
@@ -5,6 +5,9 @@ These are courses we could look at running that people might be interested in.
 ## Web
 * Bootstrap basics
 * Personal web security
+* NodeJS basics
+* ReactJS basics
+* Building a realtime chess game w/ React & Node
```

Comment:
We can experiment with AI afterwards... No reason to limit it to chess either, having a nice little online board games section on the labs website would be a cool demo